scavenger-agent-java/src/main/java/com/navercorp/scavenger/javaagent/ScavengerAgent.java

```diff
@@ -39,43 +39,18 @@ public static void premain(String args, Instrumentation inst) {
 
         new ScavengerBanner(config).printBanner(System.out);
 
-        CodeBase codeBase = null;
+        Scheduler scheduler = new Scheduler(config);
```

Comment:
The codebase scan only runs the first time, and it's awkward to have this task delegated to a scheduler object. 
Is there a reason for moving it to the scheduler?

Reply:
If scanning or uploading codebase fails, the scheduler will schedule a retry of the task. (only when async mode is off)